apps/mull-ui/src/app/components/chat-input/chat-input.tsx

```diff
@@ -2,25 +2,35 @@ import { faPaperPlane } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { IChatForm } from '@mull/types';
 import { FormikContextType } from 'formik';
-import React from 'react';
+import React, { ChangeEvent, MouseEventHandler } from 'react';
 import CustomFileUpload from '../custom-file-upload/custom-file-upload';
 import CustomTextInput from '../custom-text-input/custom-text-input';
 import './chat-input.scss';
 
 export interface ChatInputProps {
   formik: FormikContextType<IChatForm>;
+  image?: string;
+  handleFileUpload?: (event: ChangeEvent<HTMLInputElement>) => void;
+  handleCloseImage?: MouseEventHandler<HTMLImageElement>;
 }
 
-export const ChatInput = ({ formik }: ChatInputProps) => {
+export const ChatInput = ({
+  formik,
+  image,
+  handleFileUpload,
+  handleCloseImage,
+}: ChatInputProps) => {
   return (
     <form onSubmit={formik.handleSubmit} className="chat-input-container" data-testid="chat-input">
-      {/* TODO: Handle Image upload back-end */}
+      {image && (
+        <img className="chat-input-image-size" src={image} alt="" onClick={handleCloseImage} />
+      )}
```

Comment:
⚠Unfortunately, when you finish uploading a picture and it shows up on the bottom bar, the bottom bar ends up hiding part of the page, since its position absolute, and the bottom margin for the page content is still the same.
![image](https://user-images.githubusercontent.com/23544999/112737017-a0435680-8f2d-11eb-926a-96b6bcc102d3.png)

Just set the max height of the image to the max possible without stretching the bottom bar. It seems to be the easiest way to solve this problem for now.

Reply:
The positioning and size might have to be cleaned up because the preview images are now every small. 
![image](https://user-images.githubusercontent.com/44760251/112771052-c59c9700-8ff7-11eb-9ef6-ae7244dfa7a8.png)